src/parser/demonhunter/havoc/modules/Abilities.js

```diff
@@ -42,22 +41,7 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id),
           recommendedEfficiency: 0.85,
-          maxCasts: cooldown => calculateMaxCasts(cooldown, this.owner.fightDuration - combatant.getBuffUptime(SPELLS.METAMORPHOSIS_HAVOC_BUFF.id)),
-          extraSuggestion: <>This should be part of your single target rotation due to the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} icon /> talent.</>,
-        },
-      },
-      {
-        spell: SPELLS.DEATH_SWEEP, //During meta blade dance becomes this.
-        category: combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) ? Abilities.SPELL_CATEGORIES.ROTATIONAL : Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
-        cooldown: haste => 9 / (1 + haste),
-        gcd: {
-          base: 1500,
-        },
-        castEfficiency: {
-          suggestion: combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id),
-          recommendedEfficiency: 0.85,
-          maxCasts: cooldown => calculateMaxCasts(cooldown, combatant.getBuffUptime(SPELLS.METAMORPHOSIS_HAVOC_BUFF.id)),
-          extraSuggestion: <>This should be part of your single target rotation due to the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} icon /> talent.</>,
+          extraSuggestion: <>This should be part of your single target rotation due to the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} icon /> talent. This includes the <SpellLink id={SPELLS.DEATH_SWEEP.id} icon /> casts since they are the same ability and share thier cool downs.</>,
```

Comment:
```suggestion
          extraSuggestion: <>This should be part of your single target rotation due to the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} /> talent. This includes the <SpellLink id={SPELLS.DEATH_SWEEP.id} /> casts since they are the same ability and share thier cooldowns.</>,
```
`icon` should be omitted as it is default, and cooldown is written joined